VS Build Tools 2019 now fails to build Servo

```
> ./mach build -d
**********************************************************************
** Visual Studio 2019 Developer Command Prompt v16.0
** Copyright (c) 2019 Microsoft Corporation
**********************************************************************
[vcvarsall.bat] Environment initialized for: 'x64'
   Compiling mozjs_sys v0.67.1 (https://github.com/servo/mozjs?rev=8e3ac9b0228f9cd92ac2ce718cf1436860f18a8f#8e3ac9b0)
   Compiling metrics v0.0.1 (C:\Users\sasch\Documents\GitHub\servo\components\metrics)
   Compiling webvr v0.0.1 (C:\Users\sasch\Documents\GitHub\servo\components\webvr)
   Compiling webdriver_server v0.0.1 (C:\Users\sasch\Documents\GitHub\servo\components\webdriver_server)
   Compiling script_layout_interface v0.0.1 (C:\Users\sasch\Documents\GitHub\servo\components\script_layout_interface)
   Compiling layout_traits v0.0.1 (C:\Users\sasch\Documents\GitHub\servo\components\layout_traits)
   Compiling compositing v0.0.1 (C:\Users\sasch\Documents\GitHub\servo\components\compositing)
   Compiling layout_2013 v0.0.1 (C:\Users\sasch\Documents\GitHub\servo\components\layout)
   Compiling constellation v0.0.1 (C:\Users\sasch\Documents\GitHub\servo\components\constellation)
error: failed to run custom build command for `mozjs_sys v0.67.1 (https://github.com/servo/mozjs?rev=8e3ac9b0228f9cd92ac2ce718cf1436860f18a8f#8e3ac9b0)`

Caused by:
  process didn't exit successfully: `C:\Users\sasch\Documents\GitHub\servo\target\debug\build\mozjs_sys-cd52e29390809397\build-script-build` (exit code: 101)
--- stdout
[[ /c/Users/sasch/.cargo/git/checkouts/mozjs-fa11ffc7d4f1cc2d/8e3ac9b/mozjs/js/src/configure -ot /c/Users/sasch/.cargo/git/checkouts/mozjs-fa11ffc7d4f1cc2d/8e3ac9b/mozjs/js/src/configure.in ]] && touch /c/Users/sasch/.cargo/git/checkouts/mozjs-fa11ffc7d4f1cc2d/8e3ac9b/mozjs/js/src/configure || true
[[ /c/Users/sasch/.cargo/git/checkouts/mozjs-fa11ffc7d4f1cc2d/8e3ac9b/mozjs/js/src/old-configure -ot /c/Users/sasch/.cargo/git/checkouts/mozjs-fa11ffc7d4f1cc2d/8e3ac9b/mozjs/js/src/old-configure.in ]] && touch /c/Users/sasch/.cargo/git/checkouts/mozjs-fa11ffc7d4f1cc2d/8e3ac9b/mozjs/js/src/old-configure || true
! [[ /c/Users/sasch/.cargo/git/checkouts/mozjs-fa11ffc7d4f1cc2d/8e3ac9b/mozjs/js/src/configure.in -ot /c/Users/sasch/.cargo/git/checkouts/mozjs-fa11ffc7d4f1cc2d/8e3ac9b/mozjs/js/src/configure ]] && touch /c/Users/sasch/.cargo/git/checkouts/mozjs-fa11ffc7d4f1cc2d/8e3ac9b/mozjs/js/src/configure || true
! [[ /c/Users/sasch/.cargo/git/checkouts/mozjs-fa11ffc7d4f1cc2d/8e3ac9b/mozjs/js/src/old-configure.in -ot /c/Users/sasch/.cargo/git/checkouts/mozjs-fa11ffc7d4f1cc2d/8e3ac9b/mozjs/js/src/old-configure ]] && touch /c/Users/sasch/.cargo/git/checkouts/mozjs-fa11ffc7d4f1cc2d/8e3ac9b/mozjs/js/src/old-configure || true
if [[ /c/Users/sasch/.cargo/git/checkouts/mozjs-fa11ffc7d4f1cc2d/8e3ac9b/mozjs/js/src/configure -nt C:/Users/sasch/Documents/GitHub/servo/target/debug/build/mozjs_sys-f8f1fecdac8ceda2/out/config.status ]] ; then \
  cd C:/Users/sasch/Documents/GitHub/servo/target/debug/build/mozjs_sys-f8f1fecdac8ceda2/out && \
  PYTHON="C:\Python27\python.exe" MOZ_TOOLS="/" \
  CC="clang-cl.exe" CFLAGS="-MD" \
  CPP="" CPPFLAGS="" \
  CXX="clang-cl.exe" CXXFLAGS="-MD" \
  AS="" AR="" \
  STLPORT_LIBS="" \
  RUST_TARGET="x86_64-pc-windows-msvc" RUST_HOST="x86_64-pc-windows-msvc" \
  /c/Users/sasch/.cargo/git/checkouts/mozjs-fa11ffc7d4f1cc2d/8e3ac9b/mozjs/js/src/configure --disable-jemalloc --disable-js-shell --disable-tests --disable-shared-js --build-backends=RecursiveMake --enable-posix-nspr-emulation --with-visual-studio-version=2017 --target=x86_64-pc-mingw32 --host=x86_64-pc-mingw32 || (cat config.log && exit 1) ; \
fi
Reexecuting in the virtualenv
checking for vcs source checkout... no
checking for a shell... C:/Users/sasch/Documents/GitHub/servo/.servo/msvc-dependencies/moztools/3.2/msys/bin/sh.exe
checking for host system type... x86_64-pc-mingw32
checking for target system type... x86_64-pc-mingw32
checking for Python 3... c:\Users\sasch\AppData\Local\Microsoft\WindowsApps\python3.EXE (3.7.4)
checking whether cross compiling... no
checking for yasm... c:/Users/sasch/Documents/GitHub/servo/.servo/msvc-dependencies/moztools/3.2/bin/yasm.exe
checking yasm version... 1.3.0
INFO: checking for vcs source checkout...
INFO: no
INFO: checking for a shell...
INFO: C:/Users/sasch/Documents/GitHub/servo/.servo/msvc-dependencies/moztools/3.2/msys/bin/sh.exe
INFO: checking for host system type...
INFO: x86_64-pc-mingw32
INFO: checking for target system type...
INFO: x86_64-pc-mingw32
INFO: checking for Python 3...
INFO: c:\Users\sasch\AppData\Local\Microsoft\WindowsApps\python3.EXE (3.7.4)
INFO: checking whether cross compiling...
INFO: no
INFO: checking for yasm...
DEBUG: yasm: Trying yasm
INFO: c:/Users/sasch/Documents/GitHub/servo/.servo/msvc-dependencies/moztools/3.2/bin/yasm.exe
INFO: checking yasm version...
DEBUG: Executing: `c:/Users/sasch/Documents/GitHub/servo/.servo/msvc-dependencies/moztools/3.2/bin/yasm.exe --version`
INFO: 1.3.0
ERROR: Visual Studio 2017 could not be found!

--- stderr
which: python2.7: unknown command
ERROR: Visual Studio 2017 could not be found!
mozmake: *** [makefile.cargo:195: maybe-configure] Error 1
thread 'main' panicked at 'assertion failed: result.success()', C:\Users\sasch\.cargo\git\checkouts\mozjs-fa11ffc7d4f1cc2d\8e3ac9b\build.rs:128:5
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace.

warning: build failed, waiting for other jobs to finish...
error: build failed
Build FAILED in 0:00:37
```

Servo has been able to be built with VS Build Tools 2019 with its optional VC++ 2017 (thanks to #23667), but now it just fails.

--- a/components/script/Cargo.toml
+++ b/components/script/Cargo.toml
@@ -20,6 +20,7 @@ default = ["unrooted_must_root_lint"]
 webgl_backtrace = ["backtrace", "canvas_traits/webgl_backtrace"]
 js_backtrace = ["backtrace"]
 uwp = ["js/uwp"]
+vslatestinstalled = ["js/vslatestinstalled"]
 
 [build-dependencies]
 cmake = "0.1"

--- a/components/servo/Cargo.toml
+++ b/components/servo/Cargo.toml
@@ -35,6 +35,7 @@ webgl_backtrace = [
     "canvas/webgl_backtrace",
     "canvas_traits/webgl_backtrace",
 ]
+vslatestinstalled = ["script/vslatestinstalled"]
 
 [dependencies]
 background_hang_monitor = {path = "../background_hang_monitor"}

--- a/ports/glutin/Cargo.toml
+++ b/ports/glutin/Cargo.toml
@@ -43,6 +43,7 @@ profilemozjs = ["libservo/profilemozjs"]
 webdriver = ["libservo/webdriver"]
 webgl_backtrace = ["libservo/webgl_backtrace"]
 webrender_debugger = ["libservo/webrender_debugger"]
+vslatestinstalled = ["libservo/vslatestinstalled"]
 
 [target.'cfg(not(target_os = "android"))'.dependencies]
 backtrace = "0.3"

